app/views/gobierto_admin/admins/show.html.erb

```diff
@@ -3,14 +3,33 @@
 <p>
   <%= mail_to @admin.email %>
   <%= render "session_information", admin: @admin %>
+</p>
+
+<% if @admin.api_tokens.exists? %>
```

Comment:
Could you add some `<p>` to avoid this?

![Screen Shot 2020-04-24 at 08 58 52](https://user-images.githubusercontent.com/17616/80183952-0e8ae500-860a-11ea-92ae-dbc83779948b.png)